src/main/java/techcourse/myblog/service/CommentService.java

```diff
@@ -41,4 +43,19 @@ public Comment save(Comment newComment) {
                 .map(CommentResponseDto::new)
                 .collect(Collectors.toList()));
     }
+
+    public Comment findById(long commentId) {
+        return commentRepository.findById(commentId).orElseThrow(CommentNotFoundException::new);
+    }
+
+    public void deleteById(long commentId) {
+        commentRepository.deleteById(commentId);
+    }
+
+    public List<CommentResponseDto> findByArticleId(long articleId) {
```

Comment:
메서드 이름은 메서드의 정확한 동작을 구체적으로 나타내는 것이 좋겠죠?

```suggestion
    public List<CommentResponseDto> findAllByArticleId(long articleId) {
```

Reply:
반영했습니다!